### ISSUE TITLE
Campaign Start Date affected by Daylight Savings Time

### ISSUE THREAD

**Issue by jptremper** [revive-adserver/revive-adserver#634]
A client of mine was having difficulty setting up a new campaign to start on November 1, 2015. Each time they selected November 1, 2015 as the start date and saved the campaign, the start date was saved as October 31, 2015. After some digging around we realized that the issue was related to the fact that Daylight Savings Time ends on 11/1/2015.

**Comment by mbeccati** [comment]
Thanks for your report. What time zone?

**Comment by jptremper** [comment]
US Eastern (New York)

On 11/19/15, 2:10 AM, Matteo Beccati wrote:
>
> Thanks for your report. What time zone?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/revive-adserver/revive-adserver/issues/634#issuecomment-157972555>.
>

-- 

Jason Tremper
Creative Media Group, Inc.
jason@cmg.net
Phone: 908-879-9565 x134

**Comment by andrewatfornax** [comment]
Hi @jptremper - do you mean the start date saved as displayed in the UI, or as saved in the database?

**Comment by andrewatfornax** [comment]
Closing, no feedback. Please re-open if the above can be answered!

**Comment by jptremper** [comment]
I'm sorry, I never saw the question from 1/11/2016 - not sure if this is still an issue, but the problem was with how it was saved in the database.

**Comment by mbeccati** [comment]
I can confirm the bug. I've created a fresh instance with America/New_York TZ. I've created a campaign starting on Nov 4, 2018 and when open it for editing, Nov 3 shows up.

**Comment by mbeccati** [comment]
I can confirm that the campaign data is saved correctly and it will start precisely as expected:

```
postgres=# select '2018-11-04 04:00:00 UTC'::timestamptz at time zone 'America/New_York';
      timezone
---------------------
 2018-11-04 00:00:00
```

However, edititng it and saving without changing start date will save '2018-11-03 04:00:00' to the database.

**Comment by mbeccati** [comment]
Fixed in #946
